tests/valid.rs

```diff
@@ -47,7 +47,7 @@ fn run(toml: &str, json: &str) {
 
     // Assert toml == json
     let toml_json = to_json(toml.clone());
-    assert!(json == toml_json,
+    assert_eq!(json, toml_json,
```

Comment:
I was under the impression that `assert_eq!` would show the `Debug` for each operand on failure whereas `assert!` would just show the expression `json == toml_json`.